Content.Client/UserInterface/Systems/Chat/Widgets/ChatBox.xaml.cs

```diff
@@ -62,7 +86,54 @@ private void OnMessageAdded(ChatMessage msg)
 
         var color = msg.MessageColorOverride ?? msg.Channel.TextColor();
 
-        AddLine(msg.WrappedMessage, color);
+
+        // EE - Chat stacking
+        int index = _chatStackList.FindIndex(data => data.WrappedMessage == msg.WrappedMessage);
```

Comment:
I'm too tired to request this change just do it if you're bored
```suggestion
        var index = _chatStackList.FindIndex(data => data.WrappedMessage == msg.WrappedMessage);
```